build-system/tasks/check-links.js

```diff
@@ -135,7 +135,12 @@ function filterWhitelistedLinks(markdown) {
   filteredMarkdown = filteredMarkdown.replace(/<code>(.*?)<\/code>/g, '');
 
   // Dailymotion link that is marked dead in Travis CI
-  filteredMarkdown = filteredMarkdown.replace(/https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
+  filteredMarkdown = filteredMarkdown.replace(
+    /https:\/\/developer.dailymotion.com\/player#player-parameters/g, '');
+
+  // Font providers CDNs that return 404 on the root
+  filteredMarkdown = filteredMarkdown.replace(/https:\/\/fast.fonts.net/g, '');
+  filteredMarkdown = filteredMarkdown.replace(/https:\/\/fonts.googleapis.com/g, '');
```

Comment:
I just thought about this further, and there is precedent for wrapping links that are not supposed to work within backticks. See https://github.com/ampproject/amphtml/pull/10585#issuecomment-317543966 for a discussion on this. (cc @pbakaus)

If you want a uniform look across all three links listed in [this](https://github.com/robinpokorny/amphtml/blob/ffe5eff221fe7515ad1cc192669fd02529e11bdf/spec/amp-html-format.md#custom-fonts) section, may I suggest that you put all of them within backticks, since they are origins and not actual URLs that need to be clicked?